Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentAreaLight.cpp

```diff
@@ -27,7 +27,8 @@ void WindowComponentAreaLight::DrawWindowContents()
 
 	if (asAreaLight)
 	{
-		const char* lightTypes[] = { "SPHERE", "TUBE" }; //TODO ADD the rest of types once there are implemented
+		AXO_TODO("ADD the rest of types once there are implemented");
```

Comment:
```suggestion
		AXO_TODO("ADD the rest of types once they are implemented");
```

not from this PR, nice to fix anyway